src/features/public/UserProfile/components/ProjectSnippet.tsx

```diff
@@ -3,7 +3,7 @@ import Sugar from 'sugar';
 import { getCountryDataBy } from '../../../../utils/countryCurrency/countryUtils';
 import getImageUrl from '../../../../utils/getImageURL';
 import styles from '../../Donations/styles/Projects.module.scss';
-import i18next from '../../../../../i18n';
+import i18next from '../../../../../i18n/server';
```

Comment:
Letting NextJS automatically choose between `client.js` and `server.js` I assumed to be a basic concept about the architecture of the i18n integration? Why have you decided to import only the server module here? Does that work?

Reply:
Please change `i18n/server` to `i18n` again.